Project/style.css

```diff
@@ -25,35 +25,44 @@
 }
 
 /*Highlight current page*/
+
 .highlighted {
   background-color: #888;
 }
 
-li.highlighted a{
-  color: red;
+li.highlighted a {
+  color: #ae31f1;
 }
 
 /* Register a new user */
 
 .register table {
-	border: 1px solid black;
-	background-color: #cfeafa;
-	width: 100%;
+  border: 1px solid black;
+  background-color: red;/* For browsers that do not support gradients */
+  background-image: linear-gradient(#08da6d, #ae31f1);/*Gradient colour*/
+  width: 100%;
 }
 
 .register thead {
-	background-color: #009bfb; /* Blue */
+  background-color: #08da6d;/* green */
 }
 
 .register .lbl {
-	text-align: right;
+  text-align: right;
 }
 
 .register td {
-	width: 50%;
+  width: 50%;
 }
 
 /* Form field width */
 input[type=text], input[type=submit], input[type=password], select {
   width: 200px;
+}
+
+body {
+  font-family: 'Atomic Age', cursive;
+  height: 1000px;
```

Comment:
I changed 900px to 1000px as the game canvas didn't fit exactly in this body size